packages/hardhat-core/package.json

```diff
@@ -99,16 +99,16 @@
   "dependencies": {
     "@ethersproject/abi": "^5.1.2",
     "@metamask/eth-sig-util": "^4.0.0",
-    "@nomicfoundation/ethereumjs-block": "^4.0.0",
-    "@nomicfoundation/ethereumjs-blockchain": "^6.0.0",
-    "@nomicfoundation/ethereumjs-common": "^3.0.0",
-    "@nomicfoundation/ethereumjs-evm": "^1.0.0",
-    "@nomicfoundation/ethereumjs-rlp": "^4.0.0",
-    "@nomicfoundation/ethereumjs-statemanager": "^1.0.0",
-    "@nomicfoundation/ethereumjs-trie": "^5.0.0",
-    "@nomicfoundation/ethereumjs-tx": "^4.0.0",
-    "@nomicfoundation/ethereumjs-util": "^8.0.0",
-    "@nomicfoundation/ethereumjs-vm": "^6.0.0",
+    "@nomicfoundation/ethereumjs-block": "4.2.1",
+    "@nomicfoundation/ethereumjs-blockchain": "6.2.1",
+    "@nomicfoundation/ethereumjs-common": "3.1.1",
+    "@nomicfoundation/ethereumjs-evm": "1.3.1",
+    "@nomicfoundation/ethereumjs-rlp": "4.0.2",
+    "@nomicfoundation/ethereumjs-statemanager": "1.0.4",
+    "@nomicfoundation/ethereumjs-trie": "5.0.4",
+    "@nomicfoundation/ethereumjs-tx": "4.1.1",
+    "@nomicfoundation/ethereumjs-util": "8.0.5",
+    "@nomicfoundation/ethereumjs-vm": "6.4.1",
```

Comment:
I used pinned versions this time (I should've done that the previous time). The fork also has pinned versions in their intra-dependencies.